src/vorta/keyring/keyring_secretstorage.py

```diff
@@ -0,0 +1,22 @@
+from .keyring_backend import KeyringBackend
+import secretstorage
+
+
+class VortaSecretStorageKeyring(KeyringBackend):
+    """A wrapper for the secretstorage package to support the custom keyring backend"""
+
+    def set_password(self, service, repo_url, password):
+        connection = secretstorage.dbus_init()
+        collection = secretstorage.get_default_collection(connection)
+        attributes = {'application': service, 'repo_url': repo_url}
+        collection.create_item(f"vorta-{repo_url}", attributes, password.encode(), replace=True)
+
+    def get_password(self, service, repo_url):
+        connection = secretstorage.dbus_init()
```

Comment:
Didn't you want to call `dbus_init()` only once at the beginning due to some issue and to test if DBUS is actually there? In that case it would be in `VortaSecretStorageKeyring .__init__()`. If it fails, it falls back to the database keystore, like we do already in `vorta.utils`

Reply:
According to the [docs](https://secretstorage.readthedocs.io/en/latest/) it's called only once and then passed around. So better to set it in `__init__()` and then use it for password requests.

> `secretstorage.dbus_init() ` [...] Returns a new connection to the session bus, instance of jeepney.DBusConnection instance. This connection can then be passed to various SecretStorage functions, such as get_default_collection().